releases/release-5.0.4.md

```diff
@@ -0,0 +1,165 @@
+---
+title: TiDB 5.0.4 Release Notes
+---
+
+# TiDB 5.0.4 Release Notes
+
+Release Date: September 27, 2021
+
+TiDB version: 5.0.4
+
```

Comment:
## Feature enhancements
+ TiKV
    - Support changing CDC configs dynamically [#10685](https://github.com/tikv/tikv/pull/10685)

Reply:
## Feature enhancements
- TiDB
  - Support set tidb_enforce_mpp=1 to enforce use mpp mode. [#26382](https://github.com/pingcap/tidb/pull/26382)